src/ray/common/status.cc

```diff
@@ -53,6 +53,7 @@ namespace ray {
 #define STATUS_CODE_NOT_FOUND "NotFound"
 #define STATUS_CODE_DISCONNECTED "Disconnected"
 #define STATUS_CODE_SCHEDULING_CANCELLED "SchedulingCancelled"
+#define STATUS_CODE_ALREADY_EXISTS "AlreadyExists"
```

Comment:
(no action needed, not high priority, no need to be in this PR) Is there any chance we use string?
Macros for constants are discouraged.
Ref: effective C++ item-2.

Reply:
No objections. We can remove all macros and just put string literals into kCodeToStr, other than the STATUS_CODE_UNKNOWN which we can make a constexpr string_view.